pages/classes/cunning.md

```diff
@@ -114,3 +125,13 @@ With wit, flair, and fancy footwork you leap head first into combat, running rin
 </header>
 
 You do not need advantage to use your [Cunning Attack](#cunning-attack) against a creature when you are **Close** to it, and no other creatures are **Close** to you.
+
+<header>
+
+### Swashbuckling Action
+
+<p class="subheading">3rd-level Swashbuckler archetypes feature</p>
+
+</header>
+
+You can use the [Bonus Action]() granted by your [Cunning Action]() to make a **Cunning Acrobatics Check**, or take the [Dodge]() or [Use an Object]() actions.
```

Comment:
Comparing this with the 5E Monk's Ki features, this doesn't *appear* to be over-powered and fits well with the swashbuckler as close-combat rogue-type.

Is it boring though? Some synergy with [Daring Attack](https://github.com/grislyeye/three-meet/pull/84)?